src/settings.rs

```diff
@@ -11,10 +11,22 @@ pub(crate) struct IDRange {
     pub max: i64,
 }
 
+#[derive(Serialize, Deserialize, Debug)]
+pub(crate) enum Rule {
+    #[serde(rename = "MustRunAs")]
```

Comment:
we can drop all serde renames, they aren't needed, they set the same style as the enum variants already have.